packages/api/src/schemas/envConfig.json

```diff
@@ -32,6 +32,21 @@
           "type": "string",
           "description": "The access_as_user scope id of existing AAD app for Teams app.",
           "minLength": 1
+        },
+        "frontendDomain": {
+          "type": "string",
+          "description": "The frontend domain for redirect Url of existing AAD app for Teams app.",
+          "minLength": 1
+        },
+        "botId": {
+          "type": "string",
+          "description": "The bot id for identifier Uris of existing AAD app for Teams app.",
+          "minLength": 1
+        },
+        "botEndpoint": {
```

Comment:
add `botId` as dependency since they should be configured in pairs?

Reply:
Oh, thanks, will update